src/_data/whats-new.yml

```diff
@@ -4,8 +4,165 @@ description: This page contains recent changes that we think you'd like to know
+    and [`updateCustomerV2`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/update-customer-v2.html)
+    mutations to manage customer accounts. These new mutations require different input
+    objects than the `createCustomer` and `updateCustomer` mutations. To change a
+    customer's email address, use the new [`updateCustomerEmail`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/update-customer-email.html)
+    mutation.
+  versions: 2.4.1
+  type: New Topic
+  date: August 19, 2020
+  link: https://github.com/magento/devdocs/pull/7673
+  contributor: keharper
+  profile: https://github.com/keharper
+- description: Added GraphQL support for rewards points. Use the [`applyRewardPointsToCart`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/apply-reward-points.html)
+    and [`removeRewardPointsFromCart`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/remove-reward-points.html)
+    mutation to manage reward points within a cart. The [cart](https://devdocs.magento.com/guides/v2.4/graphql/queries/cart.html),
+    [customer](https://devdocs.magento.com/guides/v2.4/graphql/queries/customer.html),
+    and [storeConfig](https://devdocs.magento.com/guides/v2.4/graphql/queries/store-config.html)
+    queries also return information related to reward points.
+  versions: 2.4.1
+  type: New Topic
+  date: August 17, 2020
+  link: https://github.com/magento/devdocs/pull/7662
+  contributor: keharper
+  profile: https://github.com/keharper
+- description: Added GraphQL Vault support with the [Payflow Pro Vault](https://devdocs.magento.com/guides/v2.4/graphql/payment-methods/payflow-pro-vault.html)
+    payment method.
+  versions: 2.4.1
+  type: New Topic
+  date: August 14, 2020
+  link: https://github.com/magento/devdocs/pull/7714
+  contributor: keharper
+  profile: https://github.com/keharper
+- description: Added a [deprecation notice](https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html#technologies-magento-can-use)
+    for Elasticsearch 6.x.
+  versions: 2.3.6
+  type: Technical
+  date: July 15, 2020
+  link: https://github.com/magento/devdocs/pull/7512
+  contributor: jeff-matthews
+  profile: https://github.com/jeff-matthews
+- description: Added the [`subscribeEmailToNewsletter` mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/subscribe-email-to-newsletter.html)
+    to the _GraphQL Developer Guide_.
+  versions: 2.4.1
+  type: New Topic
+  date: July 6, 2020
+  link: https://github.com/magento/devdocs/pull/7422
+  contributor: atwixfirster
+  profile: https://github.com/atwixfirster
+- description: The GraphQL [`customer` query](https://devdocs.magento.com/guides/v2.4/graphql/queries/customer.html)
+    now returns detailed information about any order in the logged-in customer's order
+    history.  The query can now replicate what is displayed in the My Orders page
+    in the storefront. The `customerOrders` query has been deprecated in favor of
```

Comment:
I deleted the last sentence (about deprecation) from the whatsnew statement in the PR. Either rerun the rake job or delete it here.

Reply:
Done in https://github.com/magento/devdocs/pull/8050/commits/71400824f48a0c35261897b6a23d4a4e3bf9aea7